library/src/main/res/layout/library.xml

```diff
@@ -41,7 +41,7 @@
     />
 
   <ImageView
-    android:id="@+id/expand"
+    android:id="@id/expand"
```

Comment:
Yeah, this is definitely an AAPT1 error. If you really want to fix it, make the usages on lines 22 and 36 use the `@+id` syntax too. (And revert this change)